### PR TITLE
feat(pingcap/tidb): fail the CI build when found long time cost test cases

### LINKED ISSUE #2441: Fail the CI build when found long time cost test cases
Ref: 
- pingcap/tidb#46820

### PR DESCRIPTION
**Pull request by wuhuizuo**
Fixes #2441

- no need the `report` step anymore, also remove the container, we will send the data by cloud events in global CI server.
- currently hardcode the threshold value.
- fail in build when long time case found, contributors should fix it, make it run fast.

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

**Comment by wuhuizuo** [comment]
/hold

Should finish the tests manually.

**Comment by wuhuizuo** [comment]
testing in:
- [ ] https://do.pingcap.net/jenkins/job/pingcap/job/tidb/job/ghpr_unit_test/33146/

**Comment by purelind** [comment]
Please note, please make sure that all ut cases in the current tidb master branch can be completed within the threshold time. Otherwise, after it takes effect, all PR merges will be blocked.

**Comment by purelind** [comment]
In addition, please note that there may be a certain fluctuation range in the duration of test execution. If there are currently some test cases whose execution time is very close to the threshold, the execution of these test cases may also exceed the threshold.

File: pipelines/pingcap/tidb/latest/ghpr_unit_test.groovy
Comment (hawkingrei) [inline pipelines/pingcap/tidb/latest/ghpr_unit_test.groovy:106]: Bazel has config for it.

Comment (wuhuizuo) [reply pipelines/pingcap/tidb/latest/ghpr_unit_test.groovy:106]: It's for case not for target!

**Review by purelind — APPROVED** [review]

**Pull request merged**
